proto/scene.proto

```diff
@@ -134,25 +240,41 @@ message Light {
   */
   int32 id = 1;
   Transform transform = 3;
+  /*
+  @default (1, 1, 1)
+  Color of source light
+  */
   Color color = 4;
-  double intensity = 5;
   oneof light {
-    LightPoint point = 6; // @oneof 4 light
-    LightDirectional directional = 7; // @oneof_item
-    LightSphere sphere = 8; // @oneof_item @optional
-    LightSurrounding surrounding = 9; // @oneof_item
+    LightPoint point = 5; // @oneof 4 light
```

Comment:
It is okay for now, but in the future, we should never change the fields number.